docs/operator-manual/secret-management.md

```diff
@@ -1,29 +1,43 @@
 # Secret Management
 
-Argo CD is un-opinionated about how secrets are managed. There are many ways to do it, and there's no one-size-fits-all solution.
+There are two general ways to populate secrets when doing GitOps: on the destination cluster, or in Argo CD during 
+manifest hydration. We strongly recommend the former, as it is more secure and provides a better user experience.
```

Comment:
Maybe we need to make hydration an official feature before recommending it? Unless you don't mean the new Argo CD hydration feature...

Reply:
In this case I mean hydration in general, not the source hydrator feature

Reply:
Makes sense. My hope is that over time the industry will adopt "hydration" as the generic term for manifest transformation. But for now "generation" is probably clearer in this context.